libbz2.so.1 is missing from /usr/local/lib64

There should be a symlink from `libbz2.so.1.0` to `libbz2.so.1`.
```
$ ls -la /usr/local/lib64/libbz2.*
lrwxrwxrwx. 1 chronos chronos    15 Jul 27  2017 /usr/local/lib64/libbz2.so.1.0 -> libbz2.so.1.0.6
-rwxr-xr-x. 1 chronos chronos 69520 Aug 22 13:52 /usr/local/lib64/libbz2.so.1.0.6
```
This causes an error when trying to build Weston.
```
/usr/local/bin/ld: /lib64/libbz2.so.1: unknown type [0x13] section `.relr.dyn'
/usr/local/bin/ld: /lib64/libbz2.so.1: unknown type [0x13] section `.relr.dyn'
/usr/local/bin/ld: warning: libbz2.so.1, needed by /usr/local/lib64/libfontconfig.so.1, not found (try using -rpath or -rpath-link)
/usr/local/bin/ld: /usr/local/lib64/libfreetype.so.6: undefined reference to `BZ2_bzDecompress'
/usr/local/bin/ld: /usr/local/lib64/libfreetype.so.6: undefined reference to `BZ2_bzDecompressEnd'
/usr/local/bin/ld: /usr/local/lib64/libfreetype.so.6: undefined reference to `BZ2_bzDecompressInit'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:4686: weston-simple-im] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/usr/local/tmp/crew/4.0.0.tar.gz.dir/weston-4.0.0'
make: *** [Makefile:3570: all] Error 2
weston failed to install: `make V=0 -j4` exited with 2
```

--- a/packages/bz2.rb
+++ b/packages/bz2.rb
@@ -1,24 +1,11 @@
 require 'package'
 
 class Bz2 < Package
-  description 'bzip2 is a freely available, patent free (see below), high-quality data compressor.'
+  description 'bzip2 is a freely available, patent free, high-quality data compressor.'
   homepage 'http://www.bzip.org/'
   version '1.0.6-1'
-  source_url 'http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz'
-  source_sha256 'a2848f34fcd5d6cf47def00461fcb528a0484d8edef8208d6d2e2909dc61d9cd'
-
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/bz2-1.0.6-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/bz2-1.0.6-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/bz2-1.0.6-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/bz2-1.0.6-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '7b41f602ad83ecf36b72907ed86d739daaf8d177625f31d9205f7788e0cc9b5d',
-     armv7l: '7b41f602ad83ecf36b72907ed86d739daaf8d177625f31d9205f7788e0cc9b5d',
-       i686: '39e8e5157a5b645c7af0b65f36d765ae16304781fbdcecde1f34f15eed798633',
-     x86_64: '3808a7cba103b79efe35a7f131873b99d4253640715ab05b374db384272735eb',
-  })
+  source_url 'https://fossies.org/linux/misc/bzip2-1.0.6.tar.xz'
+  source_sha256 '4bbea71ae30a0e5a8ddcee8da750bc978a479ba11e04498d082fa65c2f8c1ad5'
 
   depends_on 'diffutils' => :build
 
@@ -35,22 +22,22 @@ class Bz2 < Package
     system "sed", "-i", "Makefile", "-e", "/ln -s/s:$(PREFIX)/bin/::"
 
     # Use PREFIX instead of DESTDIR
-    system "make", "PREFIX=#{CREW_DEST_DIR}/usr/local", "install"
+    system "make", "PREFIX=#{CREW_DEST_PREFIX}", "install"
 
     # Remove static library
-    system "rm", "#{CREW_DEST_DIR}/usr/local/lib/libbz2.a"
+    system "rm", "#{CREW_DEST_PREFIX}/lib/libbz2.a"
 
     # Install bzip2 using shared library by hand
     system "cp", "-p", "bzip2-shared", "bzip2"
-    system "cp", "-p", "bzip2", "#{CREW_DEST_DIR}/usr/local/bin/bzip2"
-    system "ln", "-sf", "bzip2", "#{CREW_DEST_DIR}/usr/local/bin/bunzip2"
-    system "ln", "-sf", "bzip2", "#{CREW_DEST_DIR}/usr/local/bin/bzcat"
+    system "cp", "-p", "bzip2", "#{CREW_DEST_PREFIX}/bin/bzip2"
+    system "ln", "-sf", "bzip2", "#{CREW_DEST_PREFIX}/bin/bunzip2"
+    system "ln", "-sf", "bzip2", "#{CREW_DEST_PREFIX}/bin/bzcat"
 
     # Install shared library by hand
-    system "mkdir", "-p", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}"
-    system "cp", "-p", "libbz2.so.1.0.6", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}"
-    system "ln", "-s", "libbz2.so.1.0.6", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}/libbz2.so.1.0"
-    system "ln", "-s", "libbz2.so.1.0", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}/libbz2.so.1"
+    system "mkdir", "-p", "#{CREW_DEST_LIB_PREFIX}"
+    system "cp", "-p", "libbz2.so.1.0.6", "#{CREW_DEST_LIB_PREFIX}"
+    system "ln", "-s", "libbz2.so.1.0.6", "#{CREW_DEST_LIB_PREFIX}/libbz2.so.1.0"
+    system "ln", "-s", "libbz2.so.1.0", "#{CREW_DEST_LIB_PREFIX}/libbz2.so.1"
   end
 
   def self.check

--- a/packages/bz2.rb
+++ b/packages/bz2.rb
@@ -50,6 +50,7 @@ class Bz2 < Package
     system "mkdir", "-p", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}"
     system "cp", "-p", "libbz2.so.1.0.6", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}"
     system "ln", "-s", "libbz2.so.1.0.6", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}/libbz2.so.1.0"
+    system "ln", "-s", "libbz2.so.1.0", "#{CREW_DEST_DIR}#{CREW_LIB_PREFIX}/libbz2.so.1"
   end
 
   def self.check

--- a/packages/bz2.rb
+++ b/packages/bz2.rb
@@ -3,7 +3,7 @@ require 'package'
 class Bz2 < Package
   description 'bzip2 is a freely available, patent free (see below), high-quality data compressor.'
   homepage 'http://www.bzip.org/'
-  version '1.0.6'
+  version '1.0.6-1'
   source_url 'http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz'
   source_sha256 'a2848f34fcd5d6cf47def00461fcb528a0484d8edef8208d6d2e2909dc61d9cd'
 

